docs/Ways-of-Working/Learning/Business-Value.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Business Value
+authors: 
+- Ed Earle
+reviewed: 
+reviewer:
+next-review: 2024-12-01
+---
+
+Every organisation produces "Business Value". The term refers to anything that supports the health of the organisation in the long term. For a commercial enterprise this usually means sustained revenue growth and profitability, although it may also be tightly linked to the organisation's mission.
+
+Ideally, all the work done by and for an organisation should enable (directly or indirectly) the creation or maintenance of business value. Work that does not is either wasteful or damaging.
+
+However! It may not always be apparent how work supports or negates the creation of value, especially where it isn't clear what business value actually it.
```

Comment:
Small typo - should be "especially where it isn't clear what business value actually **is.**"